test/indexing.jl

```diff
@@ -19,6 +19,19 @@
         end
     end
 
+    @testset "files not saved by Checkpoints.jl" begin
+        mktempdir(SystemPath) do path
+            Checkpoints.config("TestPkg.bar", path)
+            TestPkg.bar([1,2,3])
+            other_file = joinpath(path, "date=2021-01-01", "one")
```

Comment:
```suggestion
            other_file = joinpath(path, "date=2021-01-01", "other_file.csv")
```

Reply:
but it won't be a csv so that seems confusing (although of no importance i guess)

Reply:
OK. I figured a realistic extension would be good to have
`other_file.txt` ?

Reply:
maybe is good as is

Reply:
```suggestion
            other_file = joinpath(path, "date=2021-01-01", "other_file.txt")
```